### DISCUSSION TITLE
Additional CGO enabled build for mDNS resolution

### DISCUSSION THREAD

**Discussion by http403** [lrstanley/vault-unseal#40] [Ideas]
I'm using mDNS and the resolution of these needs either a separate library or enable CGO, which the latter is the easiest to do. Would anyone be interested?

**Comment by lrstanley** [comment]
Are you currently using binary releases, or docker images? May be able to add a variant that is cgo enabled, though I ideally don't want to manage a 2nd `Dockerfile` file for it.

**Reply by http403** [reply]
Binary only
